client/src/components/files/FileCard.vue

```diff
@@ -26,17 +26,23 @@
       <ion-icon :icon="ellipsisHorizontal" />
     </div>
     <div class="card-content">
-      <ion-avatar class="card-content__icons">
+      <div class="card-content-icons">
         <ion-icon
-          class="icon-item"
-          :icon="file.isFile() ? document : folder"
+          class="folder-icon"
+          :icon="folder"
+          v-if="fileIcon() === folder"
```

Comment:
Don't use `fileIcon()` as the selector. Use a function that gives you a boolean.